CustomApps/lyrics-plus/OptionsMenu.js

```diff
@@ -111,37 +113,32 @@ const TranslationMenu = react.memo(({ showTranslationButton, translatorLoaded, i
 				Spicetify.ReactComponent.Menu,
 				{},
 				react.createElement("h3", null, " Conversions"),
-				translatorLoaded || !isJapanese
```

Comment:
why this condition has been removed?

Reply:
That condition is there to toogle this popup
picture 1:
![image](https://user-images.githubusercontent.com/75513645/215347400-3412309b-3059-4f17-9687-ec38e62f0e83.png)

That `isJapanese` condition is always useless as we can't open that popup if the song is not japanese because this button will be hiden if the song is not japanese. Even in current lyrics-plus `isJapanese` is useless so forget `isJapanese` variable.
![image](https://user-images.githubusercontent.com/75513645/215347500-e695f2e5-bcb9-49b5-8579-cc2f2a06767d.png)

Now `translatorLoaded`
If the main functions of external lib is not available `translatorLoaded` will become `false` and loading screen like this will popup instead of the menu (Picture 1). If `translatorLoaded` is `true` (main functions of external lib is available) menu will appear like picture 1.
![image](https://user-images.githubusercontent.com/75513645/215347880-e79a4bea-f757-4c43-b097-9108d46b8ef3.png)

I removed `translatorLoaded` condition as lyrics plus wont inject the external lib as default. Instead it will check if convert option is enabled, if enabled then it will inject external lib at startup. so if convert option is diabled external lib will not be injected.
![image](https://user-images.githubusercontent.com/75513645/215348216-e2b8fae9-0527-4c02-b92d-66fd2fbdd1e0.png)

If a user enables convert by clicking that convert option. `onclick`(onChange) the external lib are injected and the lyrics are translated
![Untitled](https://user-images.githubusercontent.com/75513645/215348806-5e0671a5-74bb-4337-8496-5ea726b8aded.png)


Lets say if the user disables convert and closes spotify and relaunch and then opens the menu to enable it again.
If i didnt remove `translatorLoaded` condition user can never enable the covert option again as user will only see a popup that says loading as `translatorLoaded` will be `false` as the external lib will not be available as convert option is disabled

Reply:
that makes sense, my bad